environment.yml

```diff
@@ -15,6 +15,7 @@ dependencies:
   - ncurses=6.1=hfc679d8_1
   - r-base
   - r-rcpp
+  - r-lintr
```

Comment:
There should be no need to add this here, the package is already listed in esmvaltool/install/R/r_requirements.txt